assets/javascripts/discourse/templates/category-tag-autocomplete.raw.hbs

```diff
@@ -0,0 +1,13 @@
+<div class='autocomplete'>
+  <ul>
+    {{#each option in options}}
```

Comment:
This should be the new handlebars block format:  `{{#each options as |option|}}`